_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -856,6 +858,7 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
     m_caps.MbQpDataSupport         = 1;
     m_caps.TUSupport               = 73;
 
+#if (MFX_VERSION >= 1027)
```

Comment:
VAAPI is not related to MFX API. Why we have MFX version check here instead of VAAPI one? I believe these RT formats appeared in 1.2.x series (https://github.com/intel/libva/blob/v2.1-branch/va/va.h vs. https://github.com/intel/libva/blob/v2.2-branch/va/va.h), thus checks should be:
`#if VA_CHECK_VERSION(1,2,0)`
here and below.

Reply:
Done